Jova-Magazine

jova1495 opened the pull request:
PrimeraEdicion

jova1495 committed:
initial commit

jova1495 committed:
Jova-Maga-10Jun21
Second edition, Semantic HTML and some modification of CSS

jova1495 commented:
![Captura portada](https://user-images.githubusercontent.com/57199460/121560816-7ef6f000-c9dd-11eb-96fe-e3b35c374817.PNG)

crisostomoglobant commented on index.html:
```diff
@@ -4,23 +4,70 @@
-    <h3 class="header" id="header3">Hola <b>#BootcampMex</b> 2021!</h1>
-    <h4 class="header" id="header4">Hola <b>#BootcampMex</b> 2021!</h1>
+    <div class="conteiner"> <!--Body of magazine--> 
+      <form class="magForm"> <!--Space from magazine border to center--> 
```
`form ` los elementos form solo se utilizan para obtener inputs/datos del usuario. aca no tienes este caso
sugiero usar otro elemento

crisostomoglobant commented on index.html:
```diff
@@ -4,23 +4,70 @@
+    <div class="conteiner"> <!--Body of magazine--> 
+      <form class="magForm"> <!--Space from magazine border to center--> 
+        <br />
+        <!--Magazine Title and description-->
```
casi siempre no es buena practica dejar comentarios en el codigo. ya que esto se puede cubrir escribiendo un html semantico y un buen naming de variables/clases/functions etc

crisostomoglobant replied:
usando el Heading 1 y un span/paragraph me dice lo mismo que tu comentario 
`<!--Magazine Title and description-->`

crisostomoglobant commented on index.html:
```diff
@@ -4,23 +4,70 @@
+
+        <div class="about"> <!--Subtitles and descriptions from magazine--> 
+
+          <article class="about1">
```
`article` el elemento article no queda  aqui porque es ideal para pintar todo el articulo completo en un sitio. aca tu objetivo solo es pintar un listado de titulos de articulo con una pequena descripcion 

sugiero usar elementos como ul, li y a 

ya que solo muestran un listado de enlaces

crisostomoglobant commented on index.html:
```diff
@@ -4,23 +4,70 @@
+
+      </br>
+      </form>
+    </div>
```
me da la impresion que el HTML no tiene un buen formato 
puedes validar esto por fa

jova1495 replied:
Gracias, empiezo a modificar mi codigo

jova1495 committed:
Jova-Maga-11Jun21 
Third Edition:
-Replace element "form" by "div"
-Remove comments
-Replace "article" by "ul"
-html structure

For the moment is the third preview, I am still addign new features according my partner's comments.

Greetings

jova1495 commented:
![image](https://user-images.githubusercontent.com/57199460/121731396-7de5c180-cab6-11eb-9f4d-5997479f60e1.png)

crisostomoglobant commented on index.html:
```diff
@@ -1,26 +1,55 @@
+        <p class="header--description">Destination for climbers and adventure-seekers.</p>
+      </header>      
+      <section class="conteiner__about">
+        <ul class="about__block1">
```
ahora tienes un uso excesivo de elementos ul 
los elementos heading and p no pueden ser children de UL

crisostomoglobant replied:
puedes validar tu HTML en el sitio 

https://validator.w3.org/nu/#textarea

![image](https://user-images.githubusercontent.com/81988661/121737291-45e27c80-cabe-11eb-8f85-ebb8c7db9d91.png)
